src/main/java/org/codehaus/plexus/archiver/AbstractUnArchiver.java

```diff
@@ -351,7 +351,7 @@ else if ( isDirectory )
 
             if ( !isIgnorePermissions() && mode != null && !isDirectory )
             {
-                ArchiveEntryUtils.chmod( f, mode, getLogger(), isUseJvmChmod() );
```

Comment:
Yes, it was used to log the output of the 'chmod' command, so it is no
longer needed.